src/app/caesar-portal/caesar-portal-routing.module.ts

```diff
@@ -0,0 +1,15 @@
+import { Routes, RouterModule } from '@angular/router';
+ 
+import { CaesarHomeComponent } from './caesar-home/caesar-home.component';
+import { CaesarLoginComponent } from './caesar-login/caesar-login.component';
+import { LoginGuard } from './caesar-login/guard/caesar-login.guard';
+ 
+const appRoutes: Routes = [
+    { path: '', component: CaesarHomeComponent, canActivate: [LoginGuard] },
+    { path: 'log', component: CaesarLoginComponent },
```

Comment:
if I write ‘/login’ - it will be the same as url request to server and because of it when I update the code and after webpack changed there is such text on the screen:

Whitelabel Error Page
This application has no explicit mapping for /error, so you are seeing this as a fallback.
Fri Aug 18 01:42:33 EEST 2017
There was an unexpected error (type=Not Found, status=404).
No message available

If urls are different, it’s ok. Firstly they were the same but after this kind of problem I decided to change the url of page.